.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          command: npm install -g npm@latest
+          command: sudo npm install -g npm@latest
```

Comment:
global package location is no longer writable by the default user as of `cimg/node:16.17-browsers`